geopandas/tests/test_explore.py

```diff
@@ -797,3 +797,26 @@ def test_multiple_geoseries(self):
         gdf["centroid"] = gdf.centroid
 
         gdf.explore()
+
+    def test_map_kwds(self):
+        def check():
+            out_str = self._fetch_map_string(m)
+            assert "zoomControl:false" in out_str
+            assert "dragging:false" in out_str
+            assert "scrollWheelZoom:false" in out_str
+
+        # check that folium and leaflet Map() parameters can be passed
+        m = self.world.explore(
+            zoom_control=False, map_kwds=dict(dragging=False, scrollWheelZoom=False)
+        )
+        check()
+        # named kwarg takes priority over map_kwds
+        m = self.world.explore(
+            zoom_control=False,
+            map_kwds=dict(dragging=False, scrollWheelZoom=False, zoom_control=True),
+        )
+        check()
+        m = self.world.explore(
+            map_kwds=dict(dragging=False, scrollWheelZoom=False, zoom_control=False)
+        )
+        check()
```

Comment:
```suggestion
        with pytest.raises(
            ValueError, match="'zoom_control' cannot be specified in 'map_kwds'"
        ):
            self.world.explore(
                map_kwds=dict(dragging=False, scrollWheelZoom=False, zoom_control=False)
            )
```

With the change above, these tests can go and we can have one checkin the error instead.